prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -432,19 +431,13 @@ def generate_prediction_for_one_outcome(
                 for s, a in previous_scenarios_and_answers
             )
 
-        try:
-            output: ProbabilisticAnswer = crew.kickoff(inputs=inputs)
-        except (APIError, HTTPError) as e:
-            logger.error(
-                f"Could not retrieve response from the model provider because of {e}"
-            )
-            return None
+        output: ProbabilisticAnswer = crew.kickoff(inputs=inputs)
 
         if (
             task_research_one_outcome.tools_errors > 0
             or task_create_probability_for_one_outcome.tools_errors > 0
         ):
-            logger.error(
+            logger.warning(
```

Comment:
There are many subpredictions in this agent; log only as a warning because it's quite likely that from 10-20 subpredictions, at least one will fail. Error out only if >50% of them fail (see above).